ghost/core/test/e2e-server/services/milestones.test.js

```diff
@@ -206,29 +225,42 @@ describe('Milestones Service', function () {
 
         await createFreeMembers(10, 1);
         await createMemberWithSubscription('month', 1000, 'usd', '2023-01-10');
-        const result = await milestonesService.initAndRun();
+        const resultPromise = milestonesService.initAndRun(10);
+
+        await clock.tickAsync(10);
+        const result = await resultPromise;
 
         assert(result.members.value === 20);
         assert(result.members.emailSentAt === null);
         assert(result.arr.value === 150);
         assert(result.arr.emailSentAt === null);
+
+        assert(loggingStub.called);
     });
 
     it('Does not run when milestoneEmails labs flag is not set', async function () {
         mockManager.mockLabsDisabled('milestoneEmails');
 
-        const result = await milestonesService.initAndRun();
+        const resultPromise = milestonesService.initAndRun(10);
+        await clock.tickAsync(10);
+        const result = await resultPromise;
         assert(result === undefined);
+
+        assert(loggingStub.called);
```

Comment:
No idea why, but the stub returns a call count much MUCH higher than I anticipate and also am receiving when running it locally 🤔